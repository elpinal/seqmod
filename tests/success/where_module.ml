{
  val x = ()

  signature S = {
    module M : {type t}
  } where module M = {type t = unit val y = x}
}
