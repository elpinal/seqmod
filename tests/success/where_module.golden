purity: pure
âˆƒ!5.{S={M:{t(=unit):Type}}, x(=!5):unit}
